docs-source/getting-started/building-regoth.rst

```diff
@@ -0,0 +1,154 @@
+all platforms, CMake version 3.13 is required.
+
+REGoth uses `Vcpkg <https://github.com/microsoft/vcpkg>`_ internally.  To make building as seamless
+as possible, the build script will take care of getting a copy of Vcpkg, bootstrap it and install
+the required packages.  All this happens inside the ``build`` directory so it doesn't clutter your
+system with stuff you don't want.
+
+.. note::
+
+  If you want to use your own copy of Vcpkg, just pass the toolchain file as usual.
+
+.. note::
+
+  If you don't want to have anything to do with Vcpkg, you can also choose to supply the required
+  packages yourself.  To do this, you need to pass ``-DSKIP_AUTOMATE_VCPKG=ON`` to CMake.  Now
+  you're completely on your own to gather the required dependencies for REGoth.
+
+
+Linux
+-----
+
+
+System packages
+~~~~~~~~~~~~~~~
+
+For **Debian** (Buster or newer), **Ubuntu** (bionic or newer), or derivatives, install the
+following packages:
+
+.. code-block:: sh
+
+  # Required tooling packages.
+  $ sudo apt install git cmake make gcc
+
+  # Required dependencies.
+  $ sudo apt install libx11-dev libxcursor-dev libxi-dev libicu-dev libgl1-mesa-dev \
+  libglu1-mesa-dev freeglut3-dev libphysfs-dev libsquish-dev
+
+  # Development dependencies (optional).
+  $ sudo apt install doxygen
+
+For **Fedora**, install the following packages:
+
+.. code-block:: sh
+
+  # Required tooling packages.
+  $ sudo dnf install git cmake make gcc
+
+  # Required dependencies.
+  $ sudo dnf install libX11-devel libXcursor-devel libXi-devel libicu-devel mesa-libGL-devel \
+  mesa-libGLU freeglut-devel physfs-devel libsquish-devel libXrandr-devel
+
+  # Development dependencies (optional).
+  $ sudo dnf install doxygen
+
+For **Arch Linux**, **Manjaro**, or derivatives, install the following packages:
+
+.. code-block:: sh
+
+  # Required tooling packages.
+  $ sudo pacman -S git cmake make gcc
+
+  # Required dependencies.
+  $ sudo pacman -S physfs libx11 libxcursor libxi icu mesa freeglut libxrandr
+
+  # Development dependencies (optional).
+  $ sudo pacman -S doxygen
+
+.. note:: If a required package is missing, or you managed to compile REGoth on a distribution not
+  explicitly listed above, we would *greatly* appreciate a pull request or GitHub issue with this
+  information.
+
+
+Building
+~~~~~~~~
+
+First we need to fetch the source code and prepare our build environment:
+
+.. code-block:: sh
+
+  # First, clone the Git repository together with its submodules.
+  $ git clone --recursive https://github.com/REGoth-project/REGoth-bs.git
+  $ cd REGoth-bs
+
+  # Prepare the build directory.
+  $ mkdir build
+  $ cd build
+
+  # Run CMake.
+  $ cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
+
+Now we can build the project (if you want to use less cores than available, replace ``$(nproc)``
+with the desired number.  You might want to do this if you have little RAM, expect up to 2GB of RAM
+usage per core):
+
+.. code:: sh
```

Comment:
Missing syntax highlighting. Should probably use `.. code-block:: sh` instead of `.. code:: sh`.

Reply:
My bad, of course!